public/v2/apps/odoo.json

```diff
@@ -0,0 +1,70 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://hub.docker.com/_/odoo?tab=description",
+    "description": "Odoo, formerly known as OpenERP, is a suite of open-source business apps written in Python and released under the AGPL license. This suite of applications covers all business needs, from Website/Ecommerce down to manufacturing, inventory and accounting, all seamlessly integrated. It is the first time ever a software editor managed to reach such a functional coverage. Odoo is the most installed business software in the world. Odoo is used by 2.000.000 users worldwide ranging from very small companies (1 user) to very large ones (300 000 users)",
```

Comment:
Description is too long and messes up with the UI - let's reduce it to 150 char or less. Move the extra parts to instructions.start.